dask/dataframe/io/tests/test_parquet.py

```diff
@@ -187,12 +187,13 @@ def test_read_glob(tmpdir, write_engine, read_engine):
 
     # Infer divisions for engines/versions that support it
 
-    ddf2 = dd.read_parquet(os.path.join(fn, '*'), engine=read_engine,
```

Comment:
For the old one, were we using the metadata file? Should your tests using just the individual files be in addition to the old test, rather than in place of them?

Reply:
The only change here, was that the set of data-files shouldn't include the _common_metadata file by mistake.

Reply:
So with a directory

```
_common_metadata
part.0.parquet
part.1.parquet
```

`_common_metadata` is ignored when it's captured by the glob? Is that documented (understood that it's not immediately relevant to this PR)?

Reply:
It works fine, you get an empty partition for the non-data file, just would complicate the test a little, I feel this is cleaner. The case where there is no _common_metadata either may be more common.